deps/openssl/config/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04
+FROM ubuntu@sha256:b39db7fc56971aac21dee02187e898db759c4f26b9b27b1d80b6ad32ff330c76
```

Comment:
Does Docker Hub guarantee that untagged images are kept for a long time?

Reply:
Yeah Docker guarantees it, but the registry may still decide to prune images that aren't tagged after some time

Reply:
There are images from Node since v4. (7 years ago)

Ref: https://hub.docker.com/layers/library/node/4.2.1/images/sha256-62bee22f33eb403f77528b968b23f1e380c150e14d451116d5d59563950aad44?context=explore

I mean, yeah. They may decide it, but it is a very remote possibility, isn't it?

Reply:
I'm not sure if we should use it for Dockerfiles. At the very least you should add a comment telling which version this commit hash is.

Reply:
Got it. I'll revert the Dockerfile changes for now since we still need to figure out whether it is worth it or not.

Reply:
You'd also lose the security patching that the official images receives if you pin to a hash